humans/management/commands/check_users.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management.base import BaseCommand
```

Comment:
This file used to be called `check.py` but it conflicted with core Django command `check`, that is invoked internally when doing `./manage.py test`.